flips/20220725-cadence-borrowContract.md

```diff
@@ -0,0 +1,75 @@
+# borrowContract function for PublicAccount
+
+| Status        | Proposed    												  |
+:-------------- |:------------------------------------------------------------|
+| **FLIP #**    | [NNN](https://github.com/onflow/flow/pull/NNN)  	 		  |
+| **Author(s)** | Deniz Mert Edincik (deniz@edincik.com)  					  |
+|				| Austin Kline (austin@flowty.io)  							  | 
+| **Sponsor**   | 														      |
+| **Updated**   | 2022-07-25                                    			  |
+
+## Objective
+
+This proposed change will allow dynamic import of contracts from an address, by
+using a well known `borrow` dynamic.
```

Comment:
The objective should focus on the goal, not the *how*. Move the second part to the "Design Proposal" section

```suggestion
This proposal will allow Cadence programs to import contracts dynamically from an address.
```